cluster-up/cluster/kind/common.sh

```diff
@@ -141,6 +141,28 @@ function _fix_node_labels() {
     done
 }
 
+function _get_cri_bridge_mtu() {
+  docker network inspect bridge  | grep mtu | awk '{print $2}' | sed "s/\"//g"
+}
+
+function _patch_calico_manifest_mtu() {
+  local -r calico_manifest="$1"
+  local -r mode="$2"
+  local -r cri_mtu="$3"
+
+  local -r default_calico_mtu="1440"
+
+  if [ $mode == "ipip" ]; then
+    overhead=$((20))
+  elif [ $mode == "vxlan" ];then
+    overhead=$((50))
+  fi
+
+  calico_mtu=$((cri_mtu - overhead))
+
+  sed -e "s?veth_mtu: \"$default_calico_mtu\"?veth_mtu: \"$calico_mtu\"?g" "$calico_manifest" > "$calico_manifest"
```

Comment:
You can replace in-place with `sed` and also make a backup with `-i.bak` so this would be 

```bash
sed -i.bak -e "s?veth_mtu: \"$default_calico_mtu\"?veth_mtu: \"$calico_mtu\"?g" $calico_manifest 
```